### PR TITLE
Document united

### PR DESCRIPTION
**Pull request by Neppord**
I just wanted to make a small contribution to get going

I'm fine with this getting rejected though, I have already got what I wanted from the experience

---

**Checklist:**

- [x] Added the change to the changelog's "Unreleased" section with a link to this PR and your username
- [x] Linked any existing issues or proposals that this pull request should close
- [x] Updated or added relevant documentation in the README and/or documentation directory
- [x] Added a test for the contribution (if applicable)

### CODE REVIEW

**Comment by Neppord** [comment]
I'll 
> Added the change to the changelog's "Unreleased" section with a link to this PR and your username

If this is "useful" documentation

File: src/Data/Lens/Lens/Unit.purs [hunk 1]
```diff
@@ -5,5 +5,11 @@ import Prelude
 import Data.Lens.Lens (Lens', lens)
 
 -- | There is a `Unit` in everything.
+-- | ```purescript
+-- | > view united [1,2,3]
+-- | Unit
+-- | > over united (\a -> a) [1,2,3]
```
Comment (thomashoneyman) [inline src/Data/Lens/Lens/Unit.purs]: Perhaps we show the type here to demonstrate that `\a -> a` here has to have the type `Unit -> Unit`?
```suggestion
-- | > over united (\a -> a :: Unit) [1,2,3]
```

File: src/Data/Lens/Lens/Unit.purs [hunk 2]
```diff
@@ -5,5 +5,11 @@ import Prelude
 import Data.Lens.Lens (Lens', lens)
 
 -- | There is a `Unit` in everything.
+-- | ```purescript
+-- | > view united [1,2,3]
+-- | Unit
```
Comment (thomashoneyman) [inline src/Data/Lens/Lens/Unit.purs]: We should lower-case this as it's a value, not a type
```suggestion
-- | unit
```

Comment (Neppord) [reply src/Data/Lens/Lens/Unit.purs:10]: I don't remember if I did something strange here, but I think i pasted from the repl what I got, are you sure it shouldn't be the constructor `Unit`?

**Review by thomashoneyman — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ New features:
 Bugfixes:
 
 Other improvements:
+  - Improve documentation for `united` #134 @neppord
```
Comment (thomashoneyman) [inline CHANGELOG.md]:
```suggestion
  - Improve documentation for `united` (#134 by @neppord)
```

**Pull request merged**